rkt/rkt.go

```diff
@@ -181,7 +183,9 @@ func init() {
 
 func getTabOutWithWriter(writer io.Writer) *tabwriter.Writer {
 	aTabOut := new(tabwriter.Writer)
+
```

Comment:
I'm adding a new lines to logically group statements suggested by `Effective Go` docs.